### PR TITLE
js added, created footer and skills

### PR DESCRIPTION
**Pull request by girmaye0**

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// alert('test')
+//alert('\u00A9')
```
Comment (Lee-Rice) [inline js/script.js:2]: Not a big deal, but I'd remove this commented out code before merging.

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const skills = ["Git", "GitHub", "Git Bash", "HTML", "CSS", "JavaScript", "React", "Node.js"];
+const skillsSection = document.getElementById("skills");
+const skillsList = skillsSection.querySelector("ul");
+
+for (let i = 0; i < skills.length; i++) {
+  const skill = skills[i];
+  const listItem = document.createElement("li");
+  listItem.textContent = skill; // Add the skill text to the list item
+  skillsList.appendChild(listItem);
```
Comment (Lee-Rice) [inline js/script.js:24]: This is great. Nice work.

**Review by Lee-Rice — COMMENTED** [review]
Hi Girma,

I'm not sure what happened here, but because the link to your JavaScript file was not added to your `index.html` file, the JS you wrote doesn't appear on your html page.

I went ahead and fixed that on my side, and your JS appears to work just fine. It adds a footer and generates a list of skills.

Is it possible you just missed submitting the current html file you've been working off of?

Just a heads up, the `.js` file should be named `index.js` for this submission. Not a big deal though.

**Comment by girmaye0** [comment]
Hi Lee, the JS file  is also working on my end, too, where shall I fix ? I
created a js folder and index.js file in my directory.
Thanks for the update.
Girma

**Pull request merged**